strawberry/arguments.py

```diff
@@ -13,6 +13,21 @@
 from .utils.str_converters import to_camel_case
 
 
+class _Unset:
+    def __str__(self):
+        return ""
+
+    def __bool__(self):
+        return False
```

Comment:
Advantage of adding truthiness?

Reply:
I just moved this class around. Not sure what the point of this is. @patrick91 do you know?

Reply:
it's to make sure you can use it in `if` statements:

```
def something(a: int = UNSET):
    if a:
        return 

    ...
```

(note this check is silly but it's to demonstrate the idea :) )

Reply:
That seems like it could lead to bugs. What if `a` is `0`?

Reply:
sorry I missed this comment! that's a good point 😊 

but then that means you **always** need to check if it is unset before doing anything with it, right? I guess that's similar to using None, so I think we can remove it :)

Reply:
BTW UNSET will be changing significantly with this change anyway: https://github.com/strawberry-graphql/strawberry/issues/872